src/grandorgue/model/pipe-config/GOPipeConfig.cpp

```diff
@@ -36,6 +37,32 @@ GOPipeConfig::GOPipeConfig(
     m_IgnorePitch(-1),
     m_ReleaseTail(0) {}
 
+static const wxString WX_TUNING = wxT("Tuning");
+static const wxString WX_MANUAL_TUNING = wxT("ManualTuning");
+static const wxString WX_AUTO_TUNING_CORRECTION = wxT("AutoTuningCorrection");
+
+void GOPipeConfig::ReadTuning(
+  GOConfigReader &cfg, wxString group, wxString prefix) {
+  float legacyTuning = cfg.ReadFloat(
+    CMBSetting, group, prefix + WX_TUNING, -1800, 1800, false, 0);
+  m_ManualTuning = cfg.ReadFloat(
+    CMBSetting,
+    group,
+    prefix + WX_MANUAL_TUNING,
+    -1800,
+    1800,
+    false,
+    legacyTuning - m_PitchTuning);
+  m_AutoTuningCorrection = cfg.ReadFloat(
+    CMBSetting,
+    group,
+    prefix + WX_AUTO_TUNING_CORRECTION,
+    -1800,
+    1800,
+    false,
+    legacyTuning - m_PitchTuning);
```

Comment:
This is likely what's causing issues. For the auto tuning you should not do that calculation.

Reply:
Changed to 0